### PR TITLE
CI: install sigdb in static version of rizin

### PR DESCRIPTION
**Pull request by ret2libc**

### CODE REVIEW

**Review by wargio — APPROVED** [review]

**Review by XVilka — APPROVED** [review]

**Review by wargio — COMMENTED** [review]
this breaks cutter, if cherry-picked for 0.4.0

**Comment by ret2libc** [comment]
> this breaks cutter, if cherry-picked for 0.4.0

What does it break? I did not touch the output of `rz_sign_sigdb_load_database`.

**Comment by wargio** [comment]
> > this breaks cutter, if cherry-picked for 0.4.0
> 
> What does it break? I did not touch the output of `rz_sign_sigdb_load_database`.

https://github.com/rizinorg/cutter/blob/dev/src/core/Cutter.cpp#L3241

**Comment by ret2libc** [comment]
> > > this breaks cutter, if cherry-picked for 0.4.0
> > 
> > 
> > What does it break? I did not touch the output of `rz_sign_sigdb_load_database`.
> 
> https://github.com/rizinorg/cutter/blob/dev/src/core/Cutter.cpp#L3241

Ah, right. Let me go back to the old name.

File: librz/signature/sigdb.c [hunk 1]
```diff
@@ -137,14 +137,168 @@ RZ_API RZ_OWN RzList /*<RzSigDBEntry>*/ *rz_sign_sigdb_load_database(RZ_NONNULL
...
+static bool sigdb_to_list(void *user, const void *k, const ut64 v) {
+	RzList *l = (RzList *)user;
+	rz_list_append(l, (void *)k);
+	return true;
+}
+
+/**
+ * \brief Return the signature database as a list of entries
+ */
+RZ_API RZ_OWN RzList /* RzSigDBEntry* */ *rz_sign_sigdb_list(RZ_NONNULL const RzSigDb *db) {
```
Comment (XVilka) [inline librz/signature/sigdb.c:294]: Maybe using a vector here is better?

**Review by XVilka — APPROVED** [review]
No objections.

File: librz/signature/sigdb.c (hunk 1 above)
Comment (ret2libc) [reply librz/signature/sigdb.c:294]: i don't want to change the api, as this is used in cutter as well.

**Review by wargio — APPROVED** [review]
I think this should go in 0.4.0

**Pull request merged**